### PR TITLE
feat: add German language

### PR DESCRIPTION
**Pull request by jannkiepert**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] It's right issue type on title
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description
Adds the German language translations, updating README as well.

### CODE REVIEW

**Comment by jannkiepert** [comment]
Ok I hope the new PR and commit is better than the one before. 😄

**Comment by kyuwoo-choi** [comment]
@jannkiepert  Thank you for the contribution. 🎉 
your code just looks nice!
I'll test and merge it if I come to work tomorrow.

File: src/js/extensions/table/langs.js [hunk 1]
```diff
@@ -47,4 +47,11 @@ if (i18n) {
     'Cannot change part of merged cell': '无法更改合并单元格的一部分。',
     'Cannot paste row merged cells into the table header': '无法将行合并单元格粘贴到标题中。'
   });
+
+  i18n.setLanguage(['de', 'de_DE'], {
+    'Merge cells': 'Zellen zusammenführen',
+    'Unmerge cells': 'Zusammenführen rückgängig machen',
```
Comment (kyuwoo-choi) [inline src/js/extensions/table/langs.js]: image

`Zellen zusammenführen`
`Zusammenführen rückgängig machen`
The texts is too long and has been cut off. Can you think of shorter version of it?
or Is it ok to understand as it is?

Comment (skerbis) [reply src/js/extensions/table/langs.js]: Why not: Zellen verbinden & Zellen-Verbindung aufheben?

Comment (skerbis) [reply src/js/extensions/table/langs.js]: Später heißt es ja auch verbundene Zelle

Comment (kyuwoo-choi) [reply src/js/extensions/table/langs.js]: I think Zellen-Verbindung aufheben is still too long.
I'll consider widen the context menu if there's no more option.

Comment (skerbis) [reply src/js/extensions/table/langs.js]: Perhaps: Verbindung aufheben.

**Comment by kyuwoo-choi** [comment]
Ok. I'm merging it. I'll change the context menu more widely later.

**Pull request merged**